### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Evgenb000**
[DEMO LINK](https://Evgenb000.github.io/react_article/)

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -1 +1,30 @@
-// export const Article = () => ();
+import React from 'react';
+import '../App.scss';
+
+export const Article = (props) => {
+  const { title, date, text } = props;
```
Comment (Nazarii98) [inline src/components/Article.jsx:5]: you can do the destructuring directly instead of receiving the object

**Review by Nazarii98 — CHANGES_REQUESTED** [review]
Good job! 
Change one moment

File: src/components/Article.jsx (hunk 1 above)
Comment (SerhiiKirik) [reply src/components/Article.jsx]: No, it's ok

**Review by SerhiiKirik — APPROVED** [review]
Well done 🚀

**Pull request closed without merging**